### DISCUSSION TITLE
Welcome to wanted-codestates-project-10-9 Discussions!

### DISCUSSION THREAD

**Discussion by hinyc** [hinyc/wanted-codestates-project-10-9#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by juo1221** [comment]
팀장님 처음 이미지 리스트 랜더링할때 최신순으로 정렬되도록 GridList.jsx 코드 살짝 건드렸으니 확인해주세요
